src/styles/index.scss

```diff
@@ -39,10 +41,15 @@ $font-family: 'Roboto', sans-serif;
 body {
   font-family: $font-family;
   background: $background-color;
+  margin: 0;
 }
 
+<<<<<<< HEAD:src/index.scss
```

Comment:
Just like the other files, it looks like there are git conflicts on this page. Please resolve all conflicts